pgcli/pgcompleter.py

```diff
@@ -43,6 +49,20 @@ def __init__(self, smart_completion=True, pgspecial=None):
 
         self.all_completions = set(self.keywords + self.functions)
 
+        self.suggestion_matchers = {
```

Comment:
That's true. But having class vars defined after method definitions looks weird to me though. Makes sense though